tangelo/linq/tests/test_translator_circuit.py

```diff
@@ -41,12 +42,18 @@
                    for name in one_qubit_gate_names]
 one_qubit_gates += [Gate(name, target=1) if name not in PARAMETERIZED_GATES else Gate(name, target=1, parameter=0.2)
                     for name in one_qubit_gate_names]
+clifford_one_qubit_gates = [Gate(name, target=0) if name not in PARAMETERIZED_GATES else Gate(name, target=0, parameter=np.pi)
+                   for name in one_qubit_gate_names.pop(5)[:-1]]
+clifford_one_qubit_gates += [Gate(name, target=1) if name not in PARAMETERIZED_GATES else Gate(name, target=1, parameter=-np.pi/2)
+                    for name in one_qubit_gate_names.pop(5)[:-1]]
 two_qubit_gate_names = ["CNOT", "CX", "CY", "CZ", "CPHASE", "CRZ"]
+clifford_two_qubit_gate_names = ["CNOT", "CX", "CY", "CZ"]
 two_qubit_gates = [Gate(name, target=1, control=0) if name not in PARAMETERIZED_GATES
                    else Gate(name, target=1, control=0, parameter=0.5) for name in two_qubit_gate_names]
+clifford_two_qubit_gates = [Gate(name, target=1, control=0) for name in clifford_two_qubit_gate_names]
 swap_gates = [Gate('SWAP', target=[1, 0]), Gate('CSWAP', target=[1, 2], control=0)]
 big_circuit = Circuit(init_gates + one_qubit_gates + two_qubit_gates + swap_gates + [Gate('XX', [0, 1], parameter=0.5)])
-
+clifford_big_circuit = Circuit(init_gates + clifford_one_qubit_gates + clifford_two_qubit_gates + [Gate('SWAP', target =[1, 0])])
```

Comment:
skip a line to help us visually separate all those circuits and gates from the reference results